### PR TITLE
Render Link to manifest and source repo

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**
Helm changes:
* Set the helm parameter `git.manifestRepoUrl` to render a link to the manifest repo.
* Set the helm parameter `git.sourceRepoUrl` to render a link to the source repo.

If these variables are not set, the respective links will not be rendered. We recommend to set the `sourceRepoUrl`.

We now render both the link to the source repository (displayed as "Source"), as well as the link to the manifest repo (displayed as "Manifest").

In the Release Dialog:
image

In the history page:
image

In the planned actions:
image

### CODE REVIEW

File: services/frontend-service/src/ui/utils/Links.test.tsx [hunk 1]
```diff
@@ -189,9 +191,9 @@ describe('ReleaseVersionLink', () => {
     ];
     describe.each(cases)('RendersProperly', (testcase) => {
         const getNode = () => (
-            <DisplayLink
+            <DisplayManifestLink
```
Comment (hannesg) [inline services/frontend-service/src/ui/utils/Links.test.tsx:194]: Isn't this a bit too much too link it again?

**Review by hannesg — APPROVED** [review]
Looks good so far.

File: services/frontend-service/src/ui/utils/Links.test.tsx (hunk 1 above)
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/utils/Links.test.tsx:194]: In the test? Not sure what you mean by that @hannesg 

I will add one more test to also show that the `DisplayManifestLink` renders empty if not all input is given.

Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/utils/Links.test.tsx:194]: I added more tests
